src/app/documentation/demos/code/code-highlight-snippet.ts

```diff
@@ -6,15 +6,16 @@
 import {Component} from "@angular/core";
 
 const HTML_EXAMPLE = `
+<!-- "&#123;" left curly brace / "&#125;" right curly brace -->
 <pre>
     <code clr-code-highlight="language-css">
-        .some-component {
+        .some-component &#123;
```

Comment:
@dna2github: The code highlight example does work. Not exactly sure what the issue is here. Could you please provide more details. If you are saying that the CSS is not being highlighted its because the language we are highlight is html. The pre code blocks are getting highlighted. Also, you dont have to escape/encode the `{` `}` in a string. This change will make the code snippet look like this:

![image](https://user-images.githubusercontent.com/1426805/27540911-c09ba96c-5a36-11e7-83c0-e25f46aebc37.png)